prow/github/fakegithub/fakegithub.go

```diff
@@ -1194,6 +1197,9 @@ func (f *FakeClient) TriggerGitHubWorkflow(org, repo string, id int) error {
 }
 
 func (f *FakeClient) TriggerFailedGitHubWorkflow(org, repo string, id int) error {
+	if f.TriggerFailedGitHubWorkflowError != nil {
+		return f.TriggerFailedGitHubWorkflowError
+	}
 	return nil
```

Comment:
We can probably replace this with just
```suggestion
	return f.TriggerFailedGitHubWorkflowError
```
A `nil` error is what we want to return in case of success anyway.